tests/baselines/reference/commentOnAmbientClass1.js

```diff
@@ -1,7 +1,12 @@
 //// [tests/cases/compiler/commentOnAmbientClass1.ts] ////
 
 //// [a.ts]
-/*! Keep this pinned comment */
+/*!=========
+    Keep this pinned comment
+   =========
+*/
+
+/*! Don't keep this pinned comment */
```

Comment:
the comment text in all tests says "Don't keep this comment" but it shows. is this intended?